### PR TITLE
Para alex

### PR DESCRIPTION
**Pull request by mzalexandra2020**
https://jenkins-ozzy.portnov.com/job/para-alex/1/cucumber-html-reports/overview-features.html

### CODE REVIEW

File: cypress/e2e/amartinez.feature [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        When I click Next page Button
+        Then I verify products display in the Next page
+        When I click Previous page Button
+        Then I verify products display in the Home page
+
+
+        
+
+
+    # @demo2.1 @alex2.1
```
Comment (TroyHarding) [inline cypress/e2e/amartinez.feature:33]: Remove stale scenarios

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+alex trying to GIT
```
Comment (TroyHarding) [inline README.md:1]: Delete Read me

File: cypress/e2e/amartinez.feature [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        When I click "Contact" tab
+        Then New Message pop up opens up and I can send a message
+        When I click "About Us" tab
+        Then I verify about us video
+        When I click "Log In" tab
+        Then log in modal opens up
+        When I click "Sign Up" tab
+        Then sign up modal opens up
+
+
```
Comment (TroyHarding) [inline cypress/e2e/amartinez.feature:17]: Formating

File: cypress/support/page-object/amartinez_PO.js [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    }
+
+    // loginValidation() {
+    //   cy.get(this.validUser).should("exist").should("eql", {username: "billyjoebob"});
+    //   cy.get(this.logoutButton).should("be.visible"); 
+    // }
+
+    openHomepage(){
+      cy.visit(this.homepage);
+      cy.wait(600);
```
Comment (TroyHarding) [inline cypress/support/page-object/amartinez_PO.js:59]: i think you want 6000 to be 6 seconds

File: cypress/support/page-object/amartinez_PO.js [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+      cy.get(this.placeOrderCity).type("San Antonio");
+      cy.get(this.placeOrderCC).type("4444444444444444");
+      cy.get(this.placeOrderMonth).type("12"); 
+      cy.get(this.placeOrderYear).type("2027"); 
+      cy.get(this.placeOrderPurchaseButton).click();
+    }
+
+    confirmPurchase(){
+      cy.get(this.purchaseConfirmation).should("be.visible");
+
```
Comment (TroyHarding) [inline cypress/support/page-object/amartinez_PO.js:98]: blank space

**Review by TroyHarding — CHANGES_REQUESTED** [review]
Request changes

**Review by strgleon — APPROVED** [review]
looks good

**Review by TroyHarding — APPROVED** [review]

**Pull request merged**